### DISCUSSION TITLE
New cameras suggestion

### DISCUSSION THREAD

**Discussion by maylorian** [blakeblackshear/frigate#6745] [Ask A Question]
Hi,

 I need to get some new cameras but I can't for the life of me find anything decent from hikvision or dahua in Europe. I can however find a lot of reolink cameras off of amazon.

I remember that reolink cameras have had issues with frigate in the past, does anybody know which model of reolink cams and/or NVR I need to look for? Also open to suggestions for shop that sell hikvision cameras in Europe.

Thanks

**Comment by Matzebhv** [comment]
Hi,
i have Reolink E1 Zoom for indoor use, Reolink Duo2 and Reolink RLC-833A 4K Cams for outdoor. Works fine with frigate.
Regards
Matze
